ui/src/main/webapp/src/app/components/project-list/project-list.component.ts

```diff
@@ -56,12 +54,6 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     ngOnInit():any {
         this.updateSort();
         this.getMigrationProjects();
-        this._refreshIntervalID = setInterval(() => this.getMigrationProjects(), 30000);
```

Comment:
Well if you look at half of our PRs, they have side effects. Atomizing changes to PR gives too much overhead. I am now working on Project List UI, so I do changes in Project List UI. I would understand if I was doing changes completely unrelated.